client/components/RatingGraph.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+const RatingGraph = (props) => {
+
+  return (
+    <div className="row">
+      <div className="side">
+        <div>5 star</div>
```

Comment:
This is super minor, but happened to notice this when looking up what this bar graph looked like - all of them above `1 star` is plural as `4 stars`, `5 stars`
No need to change if this was intentional.
Nice job with this one, it's a lot of content for a single area!

Reply:
Good catch! I'll make that change right now.